roles/ceph-osd/tasks/main.yml

```diff
@@ -55,7 +55,10 @@
 
 - name: create openstack pool
   ceph_pool:
-    pool_name: "{{ ceph.pool_name }}"
+    pool_name: "{{ item }}"
+  with_items:
+    - "{{ ceph.pool_name }}"
+    - "{{ ceph.vm_pool_name }}"
```

Comment:
Wouldn't this create a vm pool, even if we aren't configuring the stack to create a vm pool? Shouldn't this be its own conditional task to create a vm pool?